OWNERS

```diff
@@ -0,0 +1,13 @@
+approvers:
```

Comment:
can you put a comment here that this is used by Red Hat for our CI?

Reply:
Done